src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Views/Obsolete/DatabaseErrorPageModel.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.Utilities;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.Views
+{
+    [Obsolete("message here")]
```

Comment:
"This type is for internal use only and will be removed in a future version."

---
In reply to: [75040354](https://github.com/aspnet/Diagnostics/pull/329#discussion_r75040354) [](ancestors = 75040354)